habitat_sim/robots/fetch_robot.py

```diff
@@ -0,0 +1,159 @@
+import magnum as mn
+import numpy as np
+import pybullet as p
+
+from habitat_sim.robots.mobile_manipulator import (
+    MobileManipulator,
+    MobileManipulatorParams,
+)
+
+
+# TODO: using PyBullet for now until we support native IK
+class IkHelper:
+    def __init__(self, urdf_path: str):
+        self._arm_len = 7
+        self.urdf_path = urdf_path
+
+    def setup_sim(self):
+        self.pc_id = p.connect(p.DIRECT)
+
+        self.robo_id = p.loadURDF(
+            self.urdf_path,
+            basePosition=[0, 0, 0],
+            useFixedBase=True,
+            flags=p.URDF_USE_INERTIA_FROM_FILE,
+            physicsClientId=self.pc_id,
+        )
+
+        p.setGravity(0, 0, -9.81, physicsClientId=self.pc_id)
+        JOINT_DAMPING = 0.5
+        self.pb_link_idx = 7
+
+        for link_idx in range(15):
+            p.changeDynamics(
+                self.robo_id,
+                link_idx,
+                linearDamping=0.0,
+                angularDamping=0.0,
+                jointDamping=JOINT_DAMPING,
+                physicsClientId=self.pc_id,
+            )
+            p.changeDynamics(
+                self.robo_id, link_idx, maxJointVelocity=200, physicsClientId=self.pc_id
+            )
+
+    def set_arm_state(self, joint_pos, joint_vel=None):
+        if joint_vel is None:
+            joint_vel = np.zeros((len(joint_pos),))
+        for i in range(7):
+            p.resetJointState(
+                self.robo_id, i, joint_pos[i], joint_vel[i], physicsClientId=self.pc_id
+            )
+
+    def calc_ik(self, targ_ee):
```

Comment:
Annotate some return types and you should see a lot of these np.array calss are unnecessary.